cms/static/js/certificates/views/signatory_details.js

```diff
@@ -6,7 +6,7 @@ define([ // jshint ignore:line
     'underscore.string',
     'backbone',
     'gettext',
-    'js/utils/templates',
+    'common/js/components/utils/templates',
```

Comment:
See here for details:

https://openedx.atlassian.net/wiki/display/TNL/How+to+add+a+new+page+to+LMS+or+Studio#HowtoaddanewpagetoLMSorStudio-requirejs-textUseUnderscoreandRequireJSTextforclient-sidetemplates